examples/atari/fibonacci.p8

```diff
@@ -0,0 +1,30 @@
+%import textio
+%zeropage basicsafe
+%address $2000
```

Comment:
the reason we have to add %address is because the atari compiler target currently uses the "NONE" launcher option. Which always requires you to specify the load address manually.

I plan on overhauling this sometime in the future because "launcher" is not really covering the intent. Maybe there will be some sort of "xex" format defined by default that automatically sets the load address to $2000 for you.

Is $2000 always the load address of atari programs like this?

Reply:
No, load address can be anywhere in memory, even zeropage ;)
Besides, the Atari supports chained block loading, so you can add multiple blocks in one file.
The first header always starts with $ff $ff load-address (lo/hi) end-address (lo/hi). The next block can be chained just by adding another load/end-address pair (4 bytes).

To make things DOS friendly, there is also one RUN address block and zero or multiple INIT address blocks.
Run address block is always these six byte: $e0 $02 $e1 $02 <run address lo/hi>
Init address block is: $e2 $02 $e3 $02 <init address lo/hi>
This way you can have a program with a loader block and init address (which runs the loader) and a main program with a main block and run address (which runs the main program).

Small and modern loaders simply use the first load address as run address, so currently it already works ;)

Reply:
Oof... don't think we will be able to support all of this in prog8

Reply:
Small steps :)

I see tass64 already supports xex output with `--atari-xex`
From the example in the manual it seems to support the run address block (using *=$02e0  .addr start).
http://tass64.sourceforge.net/